src/components/chart/statistics-chart.ts

```diff
@@ -75,6 +76,8 @@ export class StatisticsChart extends LitElement {
 
   @property({ type: Boolean }) public isLoadingData = false;
 
+  @property({ type: Boolean }) public clickForMoreInfo = true;
```

Comment:
Should set this to false when we're already in a more-info popup? 

See dialogs/more-info/ha-more-info-history.ts